.github/workflows/release.yml

```diff
@@ -52,12 +54,10 @@ jobs:
           gvsbuild build
           --platform=${{ matrix.arch }}
           --vs-ver=${{ matrix.vstudio }}
-          --same-python
```

Comment:
Quick comment on reason for removal?

Reply:
Option is not supported anymore. This is now the default and the option just throws an error.